pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>ru.yandex.clickhouse</groupId>
     <artifactId>clickhouse-jdbc</artifactId>
-    <version>0.2.4</version>
+    <version>0.2.4-enqueue</version>
```

Comment:
Please, avoid changes like this, in order not to pollute master branch.

Reply:
Sorry, this was a mistake